docs/source/release/v6.13.0/framework.rst

```diff
@@ -119,44 +164,57 @@ Python
 
 New features
 ############
-- Introduced a prototype "side-by-side" help system that includes both the legacy QtHelp-based viewer and a new Python-based Help Window using an embedded web browser (QWebEngine) to display documentation within Mantid Workbench.
-- Improvements:
+- Introduced a prototype "side-by-side" help system that includes both the legacy QtHelp-based viewer and a new
```

Comment:
I assume this section is a result of the work by @darshdinger. It is not clear from this release note how to access this prototype. Is there relevant documentation that needs to be linked to from this release note? @darshdinger and @peterfpeterson can you advise?

Reply:
@peterfpeterson queried whether this should be in the release notes for v6.13 at all

Reply:
Probably not, if we're not intending to enable it this release. I'll move it over to 6.14?